sc2reader/utils.py

```diff
@@ -20,15 +20,18 @@ class DepotFile(object):
     """
 
     #: The url template for all DepotFiles
-    url_template = "https://{0}-s2-depot.classic.blizzard.com/{1}.{2}"
+    url_template = "https://{0}-s2-depot.classic.blizzard.com{1}/{2}.{3}"
 
     def __init__(self, bytes):
         #: The server the file is hosted on
         self.server = bytes[4:8].decode("utf-8").strip("\x00 ")
-
+        #: Used to make it possible to load maps from CN. This isn't needed for any other region and so is blank by default.
+        self.urlEnd = "" 
```

Comment:
```suggestion
        self.url_end = "" 
```
Python variable naming rules.

Reply:
url_suffix seems better than url_end name wise. Either way variable has been updated to url_suffix.